.github/.wordlist.txt

```diff
@@ -908,6 +908,8 @@ namespacing
 nano
 natively
 navpad
+NBU
```

Comment:
list is case insensitive, so only one instance should be required.

However at the same time nbu, otap, prb and such are not real words (ex: https://www.merriam-webster.com/dictionary/ptb). Could you use backticks quoting in docs to not make these detected as real words?

Reply:
`pyspelling` check seems to still be failing. I will check it out tomorrow.